docs/ingest.rst

```diff
@@ -0,0 +1,239 @@
+
+Ingesting Data
+==============
+
+There are a number of different options for how to ingest data into Flow:
+
+* `HTTP PUT or POST requests`
+* `Stream data over a Websocket` in either `CSV, TSV`, or `JSON` formats
+* Write data using a `Gazette Consumer <https://gazette.readthedocs.io/en/latest/consumers-concepts.html>`_ application
```

Comment:
I don't think we want to support direct writes from consumers other than Flow; just drop this bullet?